packages/client/components/ActivityLibrary/CreateNewActivity/CreateNewActivity.tsx

```diff
@@ -0,0 +1,294 @@
+import NewMeetingTeamPicker from '../../NewMeetingTeamPicker'
+import sortByTier from '../../../utils/sortByTier'
+import {useHistory} from 'react-router'
+import {AddReflectTemplateMutation$data} from '../../../__generated__/AddReflectTemplateMutation.graphql'
+import useAtmosphere from '../../../hooks/useAtmosphere'
+import useMutationProps from '../../../hooks/useMutationProps'
+import AddReflectTemplateMutation from '../../../mutations/AddReflectTemplateMutation'
+import {Threshold} from '../../../types/constEnums'
+import useRouter from '../../../hooks/useRouter'
+import {CATEGORY_ID_TO_NAME, CATEGORY_THEMES, CategoryID, DEFAULT_CARD_THEME} from '../Categories'
+import BaseButton from '../../BaseButton'
+
+const Bold = (props: ComponentPropsWithoutRef<'span'>) => {
+  const {children, className, ...rest} = props
+
+  return (
+    <span className={clsx('font-semibold text-slate-800', className)} {...rest}>
+      {children}
+    </span>
+  )
+}
+
+const CategoryTitle = (props: ComponentPropsWithoutRef<'div'>) => {
+  const {children, className, ...rest} = props
+
+  return (
+    <div
+      className={clsx('p-4 text-lg font-semibold leading-5 text-slate-700', className)}
+      {...rest}
+    >
+      {children}
+    </div>
+  )
+}
+
+type ActivityType = 'retrospective' | 'poker'
+
+type SupportedActivity = {
+  title: string
+  type: ActivityType
+  includedCategories: CategoryID[]
+  image: string
+  isEnabled: boolean
+  phases: React.ReactNode
+}
+
+const SUPPORTED_CUSTOM_ACTIVITIES: SupportedActivity[] = [
+  {
+    title: 'Process Feedback',
+    type: 'retrospective',
+    includedCategories: ['retrospective', 'feedback', 'strategy'],
+    image: newTemplate,
+    isEnabled: true,
+    phases: (
+      <>
+        <div>
+          <Bold>Add</Bold> comments
+        </div>
+        <div>
+          <Bold>Group</Bold> comments
+        </div>
+        <div>
+          <Bold>Discuss</Bold> topics
+        </div>
+        <div>
+          <Bold>Vote</Bold> on topics
+        </div>
+        <div>
+          <Bold>Create</Bold> takeaway tasks
+        </div>
+      </>
+    )
+  },
+  {
+    title: 'Estimate Items',
+    type: 'poker',
+    includedCategories: ['estimation'],
+    image: estimatedEffortTemplate,
+    isEnabled: false,
+    phases: (
+      <>
+        <div>
+          <Bold>Select</Bold> or create issues to score
+        </div>
+        <div>
+          <Bold>Vote</Bold> on 1 or many scoring dimensions
+        </div>
+        <div>
+          <Bold>Push</Bold> esimations to your backlog
+        </div>
+      </>
+    )
+  }
+]
+
+const query = graphql`
+  query CreateNewActivityQuery {
+    viewer {
+      featureFlags {
+        retrosInDisguise
+      }
+      teams {
+        id
+        tier
+        name
+        ...NewMeetingTeamPicker_selectedTeam
+        ...NewMeetingTeamPicker_teams
+      }
+      availableTemplates(first: 100) @connection(key: "ActivityLibrary_availableTemplates") {
```

Comment:
+1 we may need to bump this up to 200 or so. It breaks in testing for me since new ones don't show up 🙃